keyboards/lets_split/keymaps/bbaserdem/README.md

```diff
@@ -1,27 +1,70 @@
-# Lets Split Layout
+# Lets Split
 
 Check out [user readme](../../../../users/bbaserdem/README.md) for more info.
+This is still a WIP, but the firmware should be fine.
 
-# Usage
+Builds fine as of 2019-12-17
 
-**These commands depend on there being no other arduino connected!**
-Also udev rules can be set instead of using sudo.
-Please unplug all other usb devices.
+# Setup
+
+I am planning on I2C, using Proton C's (or ProMicro's).
+
+# Caterina
+
+The default bootloader for ProMicro.
+
+For flashing the EEPROM with the side information, flash the following.
+```
+avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -U eeprom:w:"./quantum/split_common/eeprom-lefthand.eep"
+avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -U eeprom:w:"./quantum/split_common/eeprom-righthand.eep"
+```
+
+Afterwards, the following command should work just fine for flashing.
 
-To make the hex files;
 ```
-make lets_split/rev2:bbaserdem
-make lets_split/rev2:bbaserdem_right
+make lets_split/rev2:bbaserdem:avrdude
 ```
 
+If it does not flash, the hex files can be flashed by the following command.
+
 For the left half, after plugging in and resetting; (from repo main directory)
 ```
-sudo avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -D -U flash:w:.build/lets_split_rev2_bbaserdem.hex
-sudo avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -U eeprom:w:keyboards/lets_split/eeprom-lefthand.eep
+sudo avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -D -U flash:w:"./lets_split_rev2_bbaserdem.hex"
```

Comment:
Udev rules may be a better solution than sudo here:
https://docs.qmk.fm/#/faq_build?id=linux-udev-rules